modin/core/execution/ray/common/engine_wrapper.py

```diff
@@ -53,6 +54,7 @@ class RayWrapper:
     """Mixin that provides means of running functions remotely and getting local results."""
 
     _func_cache = {}
+    _lazy_op_refs = set()
```

Comment:
Let's add comments on what these are needed for.

Reply:
Done.